modules/apps/calendar/calendar-web-test/src/testFunctional/paths/ManageCalendar.path

```diff
@@ -12,7 +12,7 @@
 <tbody>
 <tr>
 	<td>COLOR_BY_COLUMN</td>
-	<td>//tr[${key_rowNumber}]//td[contains(@class, 'lfr-color-column')]//*[contains(@style,'${key_color}')]</td>
+	<td>//td[contains(@class, 'lfr-color-column')]//*[contains(@style,'${key_color}')]</td>
```

Comment:
When you're changing a path, always consider the macros and tests that can be affected. Removing the `key_rowNumber` variable might lead to other failures. 
Also, if you make this type of change; this path will be located if any calendar shares the same color. So for example, if you have two calendars that use the same color on the page, it'll always default to the first one.

Reply:
Hello @epark18, I appreciated your feedback, I changed now.